config/routes.rb

```diff
@@ -3,12 +3,12 @@
   devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
 
   authenticated :user do
-    root to: 'targets#index'
+    root to: 'targets#new'
   end
 
   devise_scope :user do
     root to: 'devise/sessions#new'
   end
 
-  resources :targets, only: %i[new create index]
+  resources :targets, only: %i[new create index edit destroy]
```

Comment:
Not sure about this, but isn't this the same as not having the `only`?

Reply:
I dont have an update action,but i could change this to ´except: :update´

Reply:
Oh, you are right. It's fine either way, I think :+1: